packages/world/src/IERC1967.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (interfaces/IERC1967.sol)
+
+pragma solidity ^0.8.20;
```

Comment:
Should we align this with the solidity version in other files?

Reply:
Yes, we do that with the other OZ files